src/Data/Array/Accelerate/Smart.hs

```diff
@@ -145,7 +145,7 @@ import GHC.TypeLits
 -- >       xs' = use xs
 -- >       ys' = use ys
 -- >   in
--- >   fold (+) 0 ( zipWith (*) xs' ys' )
+-- >   A.fold (+) 0 (A.zipWith (*) xs' ys')
```

Comment:
I thought the idea was that we only add `A.` on the ones that conflict with the prelude. Which example below do you mean?

Reply:
> Which example below do you mean?

`mvm`

> I thought the idea was that we only add `A.` on the ones that conflict with the prelude.

That's new to me! As far as I knew, there were two conventions:

```haskell
import Data.Array.Accelerate
import qualified Prelude as P
```

and

```haskell
import qualified Data.Array.Accelerate as A
```

What are the imports in the scenario you're describing? `import Data.Array.Accelerate as A`, in addition to being super ugly in general, wouldn't even be sufficient because then normal prelude methods are suddenly ambiguous.

Reply:
I prefer without the `A.` (both in the docs and my own code)